### ISSUE TITLE
About ordinal loss 

### ISSUE THREAD

**Issue by Sy1mple** [jaygshah/Robust-Brain-Age-Prediction#1]
I cant find the ordinal loss at this repo.
Here is the state,Did you write this module yourself?  Thank you
<img width="1280" alt="a123ba994908148239a613fcadae418" src="https://github.com/user-attachments/assets/9dfea0c9-98c7-4688-8930-0f21b8c64b05" />

**Comment by jaygshah** [comment]
Thanks for pointing it out. Please rename the `order_loss.py` as `ordinal_loss.py`. I have updated the main branch as well to avoid future confusion. Let me know if you face further issues.

**Comment by Sy1mple** [comment]
What about this one? That may be agrs?
![image](https://github.com/user-attachments/assets/4004c602-b26f-40d7-ac3a-cb264ed66ad0)

**Comment by jaygshah** [comment]
Yes, please. Thanks for catching it.

**Comment by Sy1mple** [comment]
Ok,thank you and another issue.What is about this?I check the code and find that the data read from the csv file encountered an error when doing train_test_split, so I commented out the filtering code and found that it still reported an error, is it possible to change the way train_test_split. I removed the stratify parameter and ran it and at the same time found that the generated data folder did not contain any nii files.
![image](https://github.com/user-attachments/assets/1970025e-130c-4e35-b228-3ea9e1cf3a23)
![image](https://github.com/user-attachments/assets/56947219-3b00-4766-8bc2-8e654546d609)
![image](https://github.com/user-attachments/assets/cb53477a-e628-493d-8072-bd6e2f1271bc)

**Comment by jaygshah** [comment]
The `master_data.csv` in the repository only illustrates how I structured my folders and CSV files. The `FileNotFoundError` indicates you don't have actual data (nifti images) on that path, only a path to that. You can download data from their websites upon approval or use the data you might have.

The `ValueError` indicates you need at least two images per class label since you're splitting them into train/val/test sets (age folders can't be empty after splitting into sets). In the `master_data.csv,` you have only one image per rounded age label. 

Hope this addresses your concern.

**Comment by Sy1mple** [comment]
Thank you for your help.I'm searching for data.Regards.
